cmd/validation.go

```diff
@@ -101,3 +101,13 @@ func validateRole(role string) error {
 	}
 	return errors.Errorf("please use one of: %s", strings.Join(validRoles, ", "))
 }
+
+func validateDagDeploymentArgs(dagDeploymentType, nfsLocation string) error {
+	if dagDeploymentType != "image" && dagDeploymentType != "volume" {
```

Comment:
If not specified, is dagDeploymentType defaulting to "image"? I think we should do that to be backward compatible with the current CLI.

Reply:
> is dagDeploymentType defaulting to "image"?
yes

Reply:
https://github.com/astronomer/astro-cli/blob/b20828b2c5849631253366d08f66cc9a62519450/cmd/deployment.go#L125